core/tests/test_utils.py

```diff
@@ -497,17 +498,18 @@ def get_html(self, url, params=None, expect_errors=False,
         # Reference URL:
         # https://github.com/Pylons/webtest/blob/
         # bf77326420b628c9ea5431432c7e171f88c5d874/webtest/app.py#L1119 .
-        self.assertEqual(html_response.status_int, expected_status_int)
-        if not expect_errors:
-            self.assertTrue(html_response.status_int >= 200 and
-                            html_response.status_int < 400)
-        else:
-            self.assertTrue(html_response.status_int >= 400 and
-                            html_response.status_int < 600)
-        self.assertEqual(
-            html_response.content_type, 'text/html')
-
-        return html_response
+        if not can_have_multiple_responses:
```

Comment:
I think can_have_multiple_responses is a bit of a misleading variable name. I think what you're trying to convey is more like "can be an error or not", i.e. something that contradicts expect_errors being a boolean.

So I think maybe it's worth leaving this method signature as is (I am fine with changing get_html to get_response) and creating a separate method along the lines of get_response_without_checking_for_errors(). Then have the base.py test that runs through all the URLs use that latter method instead. In fact, maybe this method here could use that method too.

Reply:
done